src/amo/components/Card/index.js

```diff
@@ -63,12 +66,34 @@ export default class Card extends React.Component {
           'Card--no-footer': !footer,
         })}
       >
-        {header ? <header className="Card-header">{header}</header> : null}
+        {header ? (
+          <header
+            className={makeClassName('Card-header', {
+              'HomepageCard-header': isHomepageCard,
+            })}
+          >
+            <div className="Card-header-text">{header}</div>
+
+            {isHomepageCard && footer ? (
+              <footer className="HomepageCard-footer-in-header">
```

Comment:
Because this is in the `Card` component, the `className` should start with `Card` and not `HomepageCard`.